<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VISWA" failures="3" tests="7" name="ExamplePriority" time="0.079" errors="0" timestamp="2024-07-15T00:18:49 IST">
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login1" time="0.019">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at retryAnalyzerPackage.retryTest.Login1(retryTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Login1 -->
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login2" time="0.001">
    <skipped/>
  </testcase> <!-- Login2 -->
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login2" time="0.001">
    <skipped/>
  </testcase> <!-- Login2 -->
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login2" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at retryAnalyzerPackage.retryTest.Login2(retryTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Login2 -->
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login3" time="0.001">
    <skipped/>
  </testcase> <!-- Login3 -->
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login3" time="0.001">
    <skipped/>
  </testcase> <!-- Login3 -->
  <testcase classname="retryAnalyzerPackage.retryTest" name="Login3" time="0.0">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at retryAnalyzerPackage.retryTest.Login3(retryTest.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Login3 -->
</testsuite> <!-- ExamplePriority -->
